app/helpers/application_helper.rb

```diff
@@ -53,13 +56,15 @@ def conditional_text(variable, prefix = '', suffix = '')
   end
 
   def prison_name_for_id(nomis_id)
-    @nomis_id_to_prison ||= Rails.configuration.prison_data.inject({}) do |h, (prison_name, prison_data)|
-      if prison_data['enabled']
-        h.update(prison_data['nomis_id'] => prison_name)
-      else
-        h
+    @nomis_id_to_prison ||=
+      Rails.configuration.prison_data.
+        inject({}) do |h, (prison_name, prison_data)|
+        if prison_data['enabled']
+          h.update(prison_data['nomis_id'] => prison_name)
+        else
+          h
+        end
```

Comment:
The indentation here is confusing. But so's the code! I think it might be clearer as:

```ruby
@nomis_id_to_prison ||= Hash[
  Rails.configuration.prison_data.
    select { |_, data| data['enabled'] }.
    map { |name, data| [data['nomis_id'], name] }
]
```

If we were on an up-to-date version of Ruby, we could use `.to_h` instead of `Hash[]`. But we're not. :cry: